torchx/schedulers/kubernetes_mcad_scheduler.py

```diff
@@ -473,8 +506,13 @@ def app_to_resource(
                 replica_role,
                 service_account,
                 image_secret,
+                coscheduler_name,
+            )
+            pod.metadata.labels.update(
+                pod_labels(
+                    app, role_idx, role, replica_id, coscheduler_name, unique_app_id
```

Comment:
there's a lot of fields here. Think it's better to switch these to use `app=app, coscheduler_name=coscheduler_name, ...` to avoid accidentally passing the wrong field to the wrong param